qbft/spectest/tests/controller/decided/current_instance.go

```diff
@@ -32,8 +34,52 @@ func CurrentInstance() tests.SpecTest {
 					DecidedCnt: 1,
 					DecidedVal: testingutils.TestingQBFTFullData,
 				},
-				ControllerPostRoot: "9a5519c1584d326e7c20cb1a824af5dc7139bac2f4d1ced2ff13b5aacfe9ab33",
+				ControllerPostRoot: currentInstanceStateComparison().Register().Root(),
 			},
 		},
 	}
 }
+
+func currentInstanceStateComparison() *qbftcomparable.StateComparison {
+	identifier := []byte{1, 2, 3, 4}
+	config := testingutils.TestingConfig(testingutils.Testing4SharesSet())
```

Comment:
for readability we might want to structure this code nicer, maybe inline instantiation so it reads better

Reply:
can u please clarify?

Reply:
I think he wants to init `contr` in oneline. To me it is less important and is up to @olegshmuelov to decide what is more readable.